egs/aishell/v1/local/produce_trials.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
I believe that @danpovey now prefers python3 for new python scripts. Could you change this to read "python3" and then verify that the script still works?